src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_EffectiveViewport.cs

```diff
@@ -48,9 +48,13 @@ private Rect WindowBounds
 #else
 		private Rect WindowBounds =>
 #if HAS_UNO
+			// At the time of writing, XamlRoot.Bounds.Size doesn't match XamlRoot.Size
+			// So, we use different path in Uno vs WinAppSDK.
+			// When this is fixed, we can remove the conditional code and use XamlRoot.Size right away.
+			// Note: XamlRoot.Bounds isn't available in WinAppSDK and is internal to Uno.
```

Comment:
@MartinZikmund FYI. I'm not sure if https://github.com/unoplatform/uno/pull/16039 will fix this?

Reply:
#16039 indeed fixed it.